CP-49980: Review provider document (#40)
xenserver/terraform-provider-xenserver: docs/data-sources/pif.md

### Optional

> - `device` (String) The machine-readable name of the physical interface (PIF). (eg. `"eth0"`)

Reviewer: Maybe use "for example," instead of "eg"

- `management` (Boolean) Indicates whether the control software is listening for connections on this physical interface.

> - `network` (String) The UUID of the virtual network to which this PIF is connected.

Reviewer 2: network seems not work, thus may need to delete CA-396475

Author: this already been fixed.

### Read-Only

> - `data_items` (Attributes List) The return items of physical network interfaces. (see [below for nested schema](#nestedatt--data_items))

Reviewer: ```suggestion
- `data_items` (Attributes List) The return items of physical network interfaces. (See [below for nested schema](#nestedatt--data_items).)
```

Author: The files under docs/ are generated by tfplugindocs. The word " (see [below for nested schema](#nestedatt--data_items))" is added by tools , so we can't modify it. Same as other tf provider see https://registry.terraform.io/providers/citrix/citrix/latest/docs/resources/admin_user.

<a id="nestedatt--data_items"></a>
### Nested Schema for `data_items`

Read-Only:

- `bond_master_of` (List of String) Indicates this PIF represents the results of a bond.
- `bond_slave_of` (String) Indicates which bond this interface is part of.
- `capabilities` (List of String) Additional capabilities on the interface.
- `currently_attached` (Boolean) True if this interface is online.

> - `device` (String) The machine-readable name of the physical interface (PIF). (eg.`"eth0"`)

Reviewer: ```suggestion
- `device` (String) The machine-readable name of the physical interface (PIF). (For example, `"eth0"`)
```

> - `disallow_unplug` (Boolean) Prevent this PIF from being unplugged; set this to notify the management tool-stack that the PIF has a special use and should not be unplugged under any circumstances. (e.g. because you're running storage traffic over it)

Reviewer: ```suggestion
- `disallow_unplug` (Boolean) Prevent this PIF from being unplugged; set this to notify the management toolstack that the PIF has a special use and should not be unplugged under any circumstances. (For example, because you're running storage traffic over it.)
```

> - `dns` (String) Comma separated list of the IP addresses of the DNS servers to use.

Reviewer: ```suggestion
- `dns` (String) Comma-separated list of the IP addresses of the DNS servers to use.
```

- `ipv6` (List of String) IPv6 address.
- `ipv6_configuration_mode` (String) Sets if and how this interface gets an IPv6 address.
- `ipv6_gateway` (String) IPv6 gateway.
- `mac` (String) Ethernet MAC address of the physical interface.
- `managed` (Boolean) Indicates whether the interface is managed by [XAPI](https://github.com/xapi-project/xen-api). If it is not, then XAPI will not configure the interface, the commands PIF.plug/unplug/reconfigure_ip(v6) cannot be used, nor can the interface be bonded or have VLANs based on top through XAPI.
- `management` (Boolean) Indicates whether the control software is listening for connections on this physical interface.
- `mtu` (Number) MTU in octets.
- `netmask` (String) IP netmask.
- `network` (String) The UUID of the virtual network to which this PIF is connected.
- `other_config` (Map of String) Additional configuration.
- `pci` (String) Link to underlying PCI device.
- `physical` (Boolean) True if this represents a physical network interface.
- `primary_address_type` (String) Which protocol should define the primary address of this interface.
- `properties` (Map of String) Additional configuration properties for the interface.

> - `sriov_logical_pif_of` (List of String) Indicates which network_sriov this interface is logical of.

Reviewer: logical _what_ of?  I assume this should say "is the logical PIF of." based on the parameter name.

Author: Those description some are directly copied from xapi repo. Here I think it means "Indicates this interface is logical pif of network_sriov "

> - `sriov_physical_pif_of` (List of String) Indicates which network_sriov this interface is physical of.

Reviewer: ```suggestion
- `sriov_physical_pif_of` (List of String) Indicates which network_sriov this interface is the physical PIF of.
```
